feature-libs/organization/account-summary/root/http-interceptors/blob-error.interceptor.ts

```diff
@@ -11,16 +11,18 @@ import {
   HttpInterceptor,
   HttpRequest,
 } from '@angular/common/http';
-import { Injectable } from '@angular/core';
+import { inject, Injectable } from '@angular/core';
 import { FileReaderService } from '@spartacus/storefront';
+import { WindowRef } from '@spartacus/core';
 import { Observable, throwError } from 'rxjs';
 import { catchError, switchMap } from 'rxjs/operators';
 
 @Injectable({
   providedIn: 'root',
 })
 export class BlobErrorInterceptor implements HttpInterceptor {
-  constructor(private fileReaderService: FileReaderService) {}
+  fileReaderService = inject(FileReaderService);
+  windowRef = inject(WindowRef);
```

Comment:
let's add a `protected` access modifier as we shouldn't give a chance to use them outsize 'intercept' logic

Reply:
sure, I think we should also add `readonly`